docs/extensions/official_extensions.md

```diff
@@ -25,3 +27,25 @@ title: Official Extensions
 | [substrait](substrait) [<span class="github">GitHub</span>](https://github.com/duckdb/substrait)                      | Adds support for the Substrait integration                                         |                 |
 | [tpcds](tpcds)                                                                                                        | Adds TPC-DS data generation and query support                                      |                 |
 | [tpch](tpch)                                                                                                          | Adds TPC-H data generation and query support                                       |                 |
+
+## Default Extensions
+
+Different DuckDB clients ship a different set of extensions.
+We summarize the main distributions in the table below.
+
+<div class="narrow_table"></div>
+
+| Extension Name | CLI (duckdb.org) | CLI (Homebrew) | Python | R | Java | Julia | Node.JS
+|------|------|------|---|---|---|---|---|
+| [autocomplete](autocomplete) | yes | yes | no  | no  | no  | no  | no  |
```

Comment:
Maybe drop the `no`s, leaving the `yes`s? Would reduce the visual noise

Reply:
good point, did this